staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go

```diff
@@ -298,15 +303,34 @@ func NewCacherFromConfig(config Config) *Cacher {
 		panic("storage codec doesn't seem to match given type: " + err.Error())
 	}
 
+	var indexedTrigger *indexedTriggerFunc
+	if config.TriggerPublisherFuncs != nil {
+		// For now, we don't support multiple trigger functions defined
+		// for a given resource.
+		// So until that is fixed, we choose only a random one and log error
```

Comment:
How are triggers configured? I'd prefer we somehow reject attempts to configure the system more than one per resource. Randomly picking one and logging an error which might go unnoticed worries me.

Reply:
I opened https://github.com/kubernetes/kubernetes/pull/80160 to make that simpler - will fix once that one merges.

Reply:
Done